### PR TITLE
Update readme

### LINKED ISSUE #19: Update README
- Merge old README with the part currently added on top
- Convert syntax to ReScript
- Remove or update badges
- Point links to correct repo

### PR DESCRIPTION
**Pull request by tom-sherman**
Closes #19

[Rendered](https://github.com/tom-sherman/rescript-webapi/blob/readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,43 @@
...
 ```
 
 ## Usage
 
-See the [examples folder](https://github.com/reasonml-community/bs-webapi-incubator/blob/master/examples/)
+See the [examples folder](https://github.com/tinymce/rescript-webapi/tree/main/examples).
 
 Please only use the modules exposed through the toplevel module `Webapi`, for example `Webapi.Dom.Element`. In particular, don't use the 'flat' modules like `Webapi__Dom__Element` as these are considered private and are not guaranteed to be backwards-compatible.
 
+## License and Credits
```
Comment (tom-sherman) [inline README.md:33]: I've moved this section below installation and usage as I feel like those two sections are more important - I hope this is ok!

Comment (TheSpyder) [reply README.md:33]: I prefer to put it at the very end; it's only near the top right now because I had kept the original readme intact.

**Review by TheSpyder — APPROVED** [review]
Looks good, just move the license section and we can merge to clean up some of the confusion with this readme 😅

**Comment by tom-sherman** [comment]
@TheSpyder Done 👍

**Pull request merged**